src/views/HomeView.vue

```diff
@@ -196,6 +255,26 @@ onMounted(async () => {
   });
 });
 
+const queueWdiget = function () {
```

Comment:
net to the typo, I would rename this to make it more clear what it does.
Something like "updateNowPlayingWidget" or whatever

Reply:
Me and my "temporary" names for things 🤦‍♂️